src/jit/lowerarm.cpp

```diff
@@ -43,9 +43,62 @@ void Lowering::LowerRotate(GenTreePtr tree)
     NYI_ARM("ARM Lowering for ROL and ROR");
 }
 
-void Lowering::TreeNodeInfoInit(GenTree* stmt)
+void Lowering::TreeNodeInfoInit(GenTree* tree)
 {
-    NYI("ARM TreeNodInfoInit");
+    Compiler* compiler = comp;
+
+    unsigned      kind         = tree->OperKind();
+    TreeNodeInfo* info         = &(tree->gtLsraInfo);
+    RegisterType  registerType = TypeGet(tree);
+
+    JITDUMP("TreeNodeInfoInit for: ");
+    DISPNODE(tree);
+    JITDUMP("\n");
+
+    switch (tree->OperGet())
+    {
+        // Please add new types below "default" case except cases which can be fit into "default" case.
+
+        // TODO-ARM-Cleanup: These cases should be replaced with "default" case after
+        // other remaining types are implmented.
+        case GT_IL_OFFSET:
```

Comment:
@pavlin I see. I also understand `IL_OFFSET`is for pseudo instrumentation and they don't have any operand. However then how about lowerarm64.cpp and lowerxarch64 ? It seems that they may have same problem, or don't they use `IL_OFFSET`?